src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -419,6 +421,22 @@
     @JsonFormat(pattern = "yyyy-MM-dd")
     private LocalDate issueDate;
 
+    @CCD(
+        label = "Notes",
+        typeOverride = Collection,
+        typeParameterOverride = "CaseNote",
+        access = {CaseworkerAndSuperUserAccess.class}
+    )
+    private List<ListValue<CaseNote>> notes;
+
+    @CCD(
+        label = "Add a case note",
+        hint = "Enter note",
+        typeOverride = TextArea,
+        access = {CaseworkerAndSuperUserAccess.class}
+    )
+    private String note;
```

Comment:
Is it not possible just to add a complex type Note and the Note::getNote field to the event rather than have an extra field?

Reply:
discussed on slack